config/initializers/assets.rb

```diff
@@ -1,3 +1,5 @@
-if Rails.application.config.should_show_easy_login
-  Rails.application.config.assets.precompile += %w( admin.css public.css easy_sign_in.js )
-end
+Rails.application.config.assets.precompile += %w(
+  admin.css
+  easy_sign_in.js
```

Comment:
Sure, it's a tiny file. In this case.

Given we have 4 different enviroments (incl. test) and all are slightly different in what they require, why stop here and make the exception...?

but hey, it's just that one file... and then another one and so on.

I thought it to be good practice to be clear in what is "going out", that is all.